connectors/kafka/build.sbt

```diff
@@ -0,0 +1,82 @@
+organization := "com.github.dcshock"
+
+name := "forklift-kafka"
+
+version := "0.1"
+
+// target and Xlint cause sbt dist to fail
+javacOptions ++= Seq("-source", "1.8")//, "-target", "1.8", "-Xlint")
```

Comment:
You can still have linting on just compile with `javacOptions in (Compile, compile)`.